src/js/components/TableFooter/README.md

```diff
@@ -1,4 +1,4 @@
-## TableFooter
+## undefined
```

Comment:
Might want to look into that.

Reply:
good catch! Using `forwardRef()` requires setting `displayName`. Updated.